other/median_of_two_arrays.py

```diff
@@ -1,7 +1,22 @@
-# This code finds the median of two arrays, even if they are not sorted initially
-def find_median_arrays(nums1, nums2):
-    all_numbers = nums1 + nums2
-    all_numbers.sort()
+from typing import List
+
+
+def median_of_two_arrays(nums1: List[float], nums2: List[float]) -> float:
+    """
+    >>> median_of_two_arrays([1, 2], [3])
+    2
+    >>> median_of_two_arrays([0, -1.1], [2.5, 1])
+    0.5
+    >>> median_of_two_arrays([], [2.5, 1])
+    1.75
+    >>> median_of_two_arrays([], [0])
+    0
+    >>> median_of_two_arrays([], [])
+    Traceback (most recent call last):
+      ...
+    IndexError: list index out of range
+    """
+    all_numbers = sorted(nums1 + nums2)
     div, mod = divmod(len(all_numbers), 2)
     if mod == 1:
         a = int(len(all_numbers) / 2)
```

Comment:
```suggestion
    if mod == 1:
        return all_numbers[div]
    else:
        return (all_numbers[div] + all_numbers[div - 1]) / 2
```
(and delete the rest of the lines in the function)

I am a bit crazy so I would have written it all on one line as:
```python
     return all_numbers[div] if mod else (all_numbers[div] + all_numbers[div - 1]) / 2
```
but the four-line version is probably easier to think about.